lib/auth/auth_test.go

```diff
@@ -160,32 +160,35 @@ func newTestPack(ctx context.Context, dataDir string) (testPack, error) {
 	return p, nil
 }
 
-func TestMain(m *testing.M) {
-	utils.InitLoggerForTests()
-	os.Exit(m.Run())
-}
-
-func TestAPI(t *testing.T) { TestingT(t) }
-
-type AuthSuite struct {
+type authSuite struct {
 	testPack
 }
 
-var _ = Suite(&AuthSuite{})
+func TestAPI(t *testing.T) { TestingT(t) }
```

Comment:
I dont think this can be removed yet, it should maybe be moved though as it is still used to run the tests in github_test.go and oidc_test.go as they have not yet been converted.